### PR TITLE
Stop setting the accessibility text if a node has SCOPES_ROUTE set.

### PR DESCRIPTION
**Pull request by gspencergoog**
This keeps us from setting the text on a node if it is a SCOPES_ROUTE node, and sends the "TYPE_VIEW_FOCUSED" event when we update the semantics information and a view has the input focus.

### CODE REVIEW

File: shell/platform/android/io/flutter/view/AccessibilityBridge.java [hunk 1]
```diff
@@ -740,7 +740,9 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
         } else {
             // Setting the text directly instead of the content description
             // will replace the "checked" or "not-checked" label.
-            result.setText(semanticsNode.getValueLabelHint());
+            if (!semanticsNode.hasFlag(Flag.SCOPES_ROUTE)) {
```
Comment (goderbauer) [inline shell/platform/android/io/flutter/view/AccessibilityBridge.java]: Combine this if with the else above?

File: shell/platform/android/io/flutter/view/AccessibilityBridge.java [hunk 2]
```diff
@@ -1377,6 +1379,12 @@ void updateSemantics(@NonNull ByteBuffer buffer, @NonNull String[] strings) {
                 event.getText().add(object.label);
                 sendAccessibilityEvent(event);
             }
+
+            // If the object is the input-focused node, then tell the reader about it.
+            if (inputFocusedSemanticsNode != null && inputFocusedSemanticsNode.id == object.id) {
+                sendAccessibilityEvent(obtainAccessibilityEvent(object.id, AccessibilityEvent.TYPE_VIEW_FOCUSED));
```
Comment (goderbauer) [inline shell/platform/android/io/flutter/view/AccessibilityBridge.java]: Shouldn't we only send this if the focused node changes?

Comment (gspencergoog) [reply shell/platform/android/io/flutter/view/AccessibilityBridge.java]: Per our conversation: removed this for this PR, and I'll make another that adds it only when the focus changes.

File: shell/platform/android/io/flutter/view/AccessibilityBridge.java (hunk 1 above)
Comment (gspencergoog) [reply shell/platform/android/io/flutter/view/AccessibilityBridge.java]: Done.

**Pull request merged**